packages/jest/src/plugins/plugin.spec.ts

```diff
@@ -64,6 +66,114 @@ describe('@nx/jest/plugin', () => {
                 ],
               },
             ],
+            "metadata": {
+              "description": "Run Jest Tests",
+              "technologies": [
+                "jest",
+              ],
+            },
+            "options": {
+              "cwd": "proj",
+            },
+            "outputs": [
+              "{workspaceRoot}/coverage",
+            ],
+          },
+        },
+      }
+    `);
+  });
+
+  it('should create test-ci targets based on jest.config.ts', async () => {
+    mockJestConfig(
+      {
+        coverageDirectory: '../coverage',
+        testMatch: ['**/*.spec.ts'],
+        testPathIgnorePatterns: ['ignore.spec.ts'],
+      },
+      context
+    );
+    const nodes = await createNodesFunction(
+      'proj/jest.config.js',
+      {
+        targetName: 'test',
+        ciTargetName: 'test-ci',
+      },
+      context
+    );
+
+    expect(nodes.projects.proj).toMatchInlineSnapshot(`
+      {
+        "root": "proj",
+        "targets": {
+          "test": {
+            "cache": true,
+            "command": "jest",
+            "inputs": [
+              "default",
+              "^production",
+              {
+                "externalDependencies": [
+                  "jest",
+                ],
+              },
+            ],
+            "metadata": {
+              "description": "Run Jest Tests",
+              "technologies": [
+                "jest",
+              ],
+            },
+            "options": {
+              "cwd": "proj",
+            },
+            "outputs": [
+              "{workspaceRoot}/coverage",
+            ],
+          },
+          "test-ci": {
+            "cache": true,
+            "dependsOn": [
+              "test-ci--src/unit.spec.ts",
+            ],
+            "executor": "nx:noop",
+            "inputs": [
+              "default",
+              "^production",
+              {
+                "externalDependencies": [
+                  "jest",
+                ],
+              },
+            ],
+            "metadata": {
+              "description": "Run Jest Tests in CI",
```

Comment:
I feel like this description should be a bit more... descriptive. It would be nice if it mentioned that running it prompts the split targets to be ran in some way.